tensorflow/core/protobuf/autotuning.proto

```diff
@@ -86,3 +126,20 @@ message AutotuningLog {
 
   // Next ID: 7
 }
+
+message AutotuningExecutionPlanLog {
```

Comment:
ditto - this won't be needed if there is no `AutotuneExecutionPlanResult`

Reply:
Done.